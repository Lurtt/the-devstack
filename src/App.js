import React, { Component } from 'react'

import 'styles/normalize'
import { Example } from 'components'

class App extends Component {
  render() {
    return <Example />
  }
}

export default App
