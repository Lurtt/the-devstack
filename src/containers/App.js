import React, { Component } from 'react'
import styled from 'styled-components'

import 'styles/normalize'
import { Layout } from 'components'

class App extends Component {
  render() {
    return (
      <Layout>
        <TempContent>here is the content</TempContent>
      </Layout>
    )
  }
}

const TempContent = styled.div``

export default App
