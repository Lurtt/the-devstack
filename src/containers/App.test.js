import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import App from './App'

describe('<App />', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<App />)
  })

  it('should display correctly', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
