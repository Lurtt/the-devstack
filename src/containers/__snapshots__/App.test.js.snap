// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should display correctly 1`] = `
<Layout>
  <TempContent>
    here is the content
  </TempContent>
</Layout>
`;
